x-pack/test/functional/apps/machine_learning/anomaly_detection/saved_search_job.ts

```diff
@@ -272,7 +272,7 @@ export default function({ getService }: FtrProviderContext) {
   ];
 
   describe('saved search', function() {
-    this.tags(['smoke', 'mlqa']);
+    this.tags(['smoke', 'mlqa', 'dima']);
```

Comment:
Please remove this debug tag

Reply:
😬 https://github.com/elastic/kibana/pull/55062/commits/2ad117e80f843a33f4676d96ecda045ccc1ff2f4